lib/Rx/Operator/SkipWhileOperator.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Rx\Operator;
+
+use Rx\ObservableInterface;
+use Rx\Observer\CallbackObserver;
+use Rx\ObserverInterface;
+use Rx\SchedulerInterface;
+
+class SkipWhileOperator implements OperatorInterface
+{
+    /** @var callable */
+    private $predicate;
+
+    /** @var int */
+    private $index;
+
+    /** @var bool */
+    private $lastResult;
+
+    public function __construct(callable  $predicate)
+    {
+        $this->predicate  = $predicate;
+        $this->index      = 0;
+        $this->lastResult = false;
```

Comment:
We should close over the `$index` and `$lastResult` in `__invoke()`. There's nothing stopping a developer from using the same operator instance twice.